cif_ed.dic

```diff
@@ -0,0 +1,643 @@
+;
+    Standard uncertainty of _diffrn.precession_angle.
+;
+    _name.category_id             diffrn
+    _name.object_id               precession_angle_su
+    _name.linked_item_id          '_diffrn.precession_angle'
+    _type.purpose                 SU
+    _type.source                  Recorded
+    _type.container               Single
+    _type.contents                Real
+    _units.code                   degrees
+
+save_
+
+save_diffrn_measurement.method_precession
+
+    _definition.id                '_diffrn_measurement.method_precession'
+    _alias.definition_id          '_diffrn_measurement_method_precession'
+    _definition.update            2024-04-22
+    _description.text
+;
+    Yes or no flag indicating if the radiation beam illuminates the sample
+    at an angle that is rotated about a precession axis.
+;
+    _name.category_id             diffrn_measurement
+    _name.object_id               method_precession
+    _type.purpose                 Encode
+    _type.source                  Assigned
+    _type.container               Single
+    _type.contents                Code
+    loop_
+      _enumeration_set.state
+      _enumeration_set.detail
+         yes                      'Precession method is used.'
+         y                        'Abbreviation for "yes".'
+         no                       'Precession method is not used.'
+         n                        'Abbreviation for "no".'
+
+    _enumeration.default          no
+
+save_
+
+save_diffrn_measurement.integration
+
+    _definition.id                '_diffrn_measurement.integration'
+    _alias.definition_id          '_diffrn_measurement_integration'
+    _definition.update            2024-04-22
+    _description.text
+;
+    Code describing the technique used to sample as completely as possible
+    the accessible range of reciprocal space.
+;
+    _name.category_id             diffrn_measurement
+    _name.object_id               integration
+    _type.purpose                 Encode
+    _type.source                  Assigned
+    _type.container               Single
+    _type.contents                Code
+    loop_
+      _enumeration_set.state
+      _enumeration_set.detail
+         beam-tilt
+;
+         The radiation beam is tilted in order to collect intensities
+         in different regions of reciprocal space while the sample
+         remains stationary.
+;
+         continuous
+;
+         Synonym for "rotation" commonly used in 3D electron diffraction.
+;
+         rotation
+;
+         The detector records diffracted intensities continuously while the
+         sample is rotated. This is the usual mode for X-ray crystallography.
+;
+         stepwise
+;
+         The detector records diffracted intensities at intervals while the
+         sample is panned or tilted in discrete steps. Often used in 3D
+         electron diffraction.
+;
+
+    _enumeration.default          rotation
+
+save_
+
+save_diffrn_measurement.tracking
```

Comment:
Items with the "yes/no" values in the `CIF_CORE` dictionary often have the "_flag" postfix, so renaming the item to `_diffrn_measurement.tracking_flag` or `_diffrn_measurement.tracking_method_flag` might be clearer.

Reply:
Will include as an issue for community consideration.